tutorials/next-chat-application/pages/api/hello.js

```diff
@@ -0,0 +1,5 @@
+// Next.js API route support: https://nextjs.org/docs/api-routes/introduction
```

Comment:
Can you remove this boilerplate code please

Reply:
Done.